### ISSUE TITLE
Linux System.Xml.Linq build failure

### ISSUE THREAD

**Issue by glunardi** [hbons/SparkleShare#1809]
As reported by the CI: https://travis-ci.org/hbons/SparkleShare/jobs/343015857
on my local machine (F27) the build is failing also because of System.Xml.Linq.

```
Making all in build
Making all in m4
Making all in Sparkles
  MCS   ../bin/Sparkles.dll
./Configuration.cs(22,18): error CS0234: The type or namespace name `Linq' does not exist in the namespace `System.Xml'. Are you missing `System.Xml.Linq' assembly reference?
./Preset.cs(20,18): error CS0234: The type or namespace name `Linq' does not exist in the namespace `System.Xml'. Are you missing `System.Xml.Linq' assembly reference?
```

Quickly looking, the assembly is there:
```
$ rpm -ql mono-core | grep -i linq
/usr/lib/mono/4.5/Facades/System.Linq.Expressions.dll
/usr/lib/mono/4.5/Facades/System.Linq.Parallel.dll
/usr/lib/mono/4.5/Facades/System.Linq.Queryable.dll
/usr/lib/mono/4.5/Facades/System.Linq.dll
/usr/lib/mono/4.5/System.Xml.Linq.dll
/usr/lib/mono/gac/System.Xml.Linq
/usr/lib/mono/gac/System.Xml.Linq/4.0.0.0__b77a5c561934e089
/usr/lib/mono/gac/System.Xml.Linq/4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
/usr/lib/mono/gac/System.Xml.Linq/4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll.mdb
```

So probably just something silly.

Related to 
https://github.com/hbons/SparkleShare/commit/cddaa86dba6de562d67e4c8c26b69691e8b21507
https://github.com/hbons/SparkleShare/commit/d5a76206835fc09ab70a0f35162a4d8018598dbd

**Comment by hbons** [comment]
Noticed this too. On macOS this works fine. I can't seem to compile it even with a direct "-r:System.Xml.Linq" and the .dll in the GAC...

**Comment by hbons** [comment]
Anyway, it works with Meson now. Check out the feature/meson branch. I might just merge that if it works for you. :)

**Comment by glunardi** [comment]
Alright, might have gone way too fast over this but: 

Seems to 
`Work for me (TM) `
here.

Well done!

**Comment by hbons** [comment]
Fixed by #1808.
